<launch>
  <arg name="sync" default="false" />
  <arg name="node_name" default="voxel_grid_filter" />
  <!-- <arg name="points_topic" default="/apollo/sensor/velodyne32/PointCloud2/fusion" /> -->
  <arg name="points_topic" default="/os_cloud_node/points" />
  <arg name="output_log" default="false" />
  <arg name="leaf_size" default="1.0" />

  <node pkg="ndt_localizer" name="$(arg node_name)" type="$(arg node_name)" output="screen">
    <param name="points_topic" value="$(arg points_topic)" />
    <remap from="/points_raw" to="/sync_drivers/points_raw" if="$(arg sync)" />
    <param name="output_log" value="$(arg output_log)" />
    <param name="leaf_size" value="$(arg leaf_size)" />
  </node>
</launch>
